build/windows/installer/conf/fluent-win-ama.conf

```diff
@@ -0,0 +1,48 @@
+<source>
+  @type containerinventory
+  tag oneagent.containerInsights.CONTAINER_INVENTORY_BLOB
+  run_interval 60
+  @log_level debug
+</source>
+
+<match **CONTAINER_INVENTORY_BLOB**>
+  @type named_pipe
+  @log_level debug
+  datatype CONTAINER_INVENTORY_BLOB
+  <format>
+  @type msgpack
+  </format>
+    <buffer tag>
+    @type file
+    path /etc/amalogswindows/out_namedpipe_containerinventoryblob.buffer
+    overflow_action drop_oldest_chunk
+    chunk_limit_size 4m
+    queue_limit_length 20
+    retry_max_times 10
+    retry_wait 5s
+    retry_max_interval 5m
+    flush_interval 20s
+    flush_thread_count 1
```

Comment:
We would need to more than 1 thread count to achieve max throughput. Refer https://docs.fluentd.org/deployment/performance-tuning-single-process#use-flush_thread_count-parameter.